napari/layers/labels/_labels_mouse_bindings.py

```diff
@@ -1,12 +1,24 @@
 from ._labels_utils import interpolate_coordinates
+from ._labels_constants import Mode
 
 
 def paint(layer, event):
-    """Paint with the currently selected label."""
+    """Paint with the proper label.
+
+    Use current selected label when in paint mode,
+    and use background label when in erase mode (or right click).
+    """
+    if layer._mode == Mode.ERASE:
+        label = layer._background_label
+    # Experimental:
+    elif hasattr(event, 'button') and event.button == 2:
```

Comment:
This doesn't seem to be working for me as expected.

Actually what looks like is happening for me is this is getting triggered by `control` click, not `right` click on the mac, and that the `control` keybinding as above isn't working.

Let's hold off on the `right click` for now, and maybe we have to rethink the modifier keybinding above.

A designer will be joining the team in July and she can help us with some of these UI choices then too